lib/colonel_kurtz/block_type.ex

```diff
@@ -1,30 +1,60 @@
 defmodule ColonelKurtz.BlockType do
+  @moduledoc """
+  The BlockType module defines a macro that is used to mix in the Block Type
+  behavior. Block Types are embedded Ecto Schemas that conform to the
+  Validatable protocol.
+
+  A Block is of some type (e.g. "Image Block") and have a well-defined schema
+  for the attributes they require. That schema is specified as a dynamically
+  generated nested module within the using module and is defined by a user call
+  to the `defattributes` macro that specifies the schema for the Blocks' Content.
+  The Block Type schema embeds_one of the Content module.
```

Comment:
```suggestion
  A Block has a `type` (e.g. "image"), a list of children (`blocks`), and a well-defined schema
  with content attributes (`content`). The content schema is defined by a user call
  to the `defattributes` macro, which generates an embedded schema under the hood.
  The Block Type schema `embeds_one` of the Content module.
```

Reply:
This suggestion mixes two things together. First, I tried to tighten up the definition of a Block. Second, I tried to hide some of the implementation details for the content module and schema.

Reply:
Thank you for taking a pass at this, I like your version a lot better.